CHANGES.md

```diff
@@ -8,6 +8,7 @@ Change Log
 * Fixed calculation of end dates for moving-point csv files, which could lead to points disappearing periodically.
 * Fixed a bug that prevented fractional seconds in time-varying WMS periodicity.
 * Added the ability to the workbench UI to select the `style` to use to display a Web Map Service (WMS) layer when multiple styles are available.
+* Added the ability to the workbench UI to select from among the available dimensions of a Web Map Service (WMS) layer.
```

Comment:
Worth adding to `CHANGES.md` that the COLORSCALERANGE functionality now works with ncWMS2 servers too?